packages/drawnix/src/components/toolbar/freehand-toolbar/freehand-toolbar.tsx

```diff
@@ -0,0 +1,82 @@
+import classNames from 'classnames';
+import { Island } from '../../island';
+import Stack from '../../stack';
+import { ToolButton } from '../../tool-button';
+import {
+  EraseIcon,
+  FeltTipPenIcon,
+} from '../../icons';
+import { BoardTransforms } from '@plait/core';
+import React from 'react';
+import { BoardCreationMode, setCreationMode } from '@plait/common';
+import { FreehandShape } from '../../../plugins/freehand/type';
+import { useBoard } from '@plait-board/react-board';
+import { splitRows } from '../../../utils/common';
+import {
+    DrawnixPointerType,
+} from '../../../hooks/use-drawnix';
+
+export interface FreehandProps {
+    title: string;
+    icon: React.ReactNode;
+    pointer: DrawnixPointerType;
+}
+
+const FREEHANDS: FreehandProps[] = [
+  {
+      icon: FeltTipPenIcon,
+      pointer: FreehandShape.feltTipPen,
+      title: 'toolbar.pen',
+    },
+    {
+      icon: EraseIcon,
+      pointer: FreehandShape.eraser,
+      title: 'toolbar.eraser',
+    },
+];
+
+const ROW_FREEHANDS = splitRows(FREEHANDS, 5);
+
+export type FreehandPickerProps = {
+  onPointerUp: (pointer: DrawnixPointerType) => void;
+};
+
+export const FreehandPicker: React.FC<FreehandPickerProps> = ({
```

Comment:
Could we define the component as `FreehandPanel`?

The component name is `FreehandPicker` and file name is `freehand-toolbar.tsx`, it's not uniform.